### PR TITLE
Simplify FROM, TO logic

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: aboki [hunk 1]
```diff
@@ -34,7 +34,6 @@ from bs4 import BeautifulSoup
 from docopt import docopt
 from requests import get, post
 
-
```
Comment (akinjide) [inline aboki:37]: The extra whitespaces for readability

**Comment by akinjide** [comment]
Hi @cclauss Awesome stuff.  🤙
I have updated the documentation and added few changes at my end, would it be fine to update your PR.

**Comment by akinjide** [comment]
@cclauss you can update your PR. ✅

**Comment by cclauss** [comment]
Please merge but do not use...  It contains syntax errors.

**Comment by akinjide** [comment]
I can't merge if it contains syntax errors, but I can review and update the errors if its fine by you?

**Comment by akinjide** [comment]
Can you also squash the commits. Thanks.

**Comment by cclauss** [comment]
Can you please turn on this option?  https://help.github.com/articles/configuring-commit-squashing-for-pull-requests/

**Comment by cclauss** [comment]
Once that is done, you should be able to do a "squash merge".

**Comment by akinjide** [comment]
Cool! Done.

**Comment by cclauss** [comment]
So, are you be able to do a "squash merge"?

**Comment by akinjide** [comment]
I see the "squash merge" option, but we need to work on your branch before merging

**Pull request closed without merging**